src/content/docs/infrastructure/install-infrastructure-agent/linux-installation/enable-logs-monitored-containers.mdx

```diff
@@ -0,0 +1,103 @@
+---
+title: Enable logs for instrumented containers
+tags:
+  - Infrastructure
+  - Install the infrastructure agent
+  - Linux installation
+metaDescription: New Relic's infrastructure monitoring agent automatically instruments container runtimes to collect container metrics and metadata.
+freshnessValidatedDate: never
+---
+
+import infrastructureAddNamesLogs from 'images/infrastructure_screenshot-crop_add-names-logs.webp'
+
+import infrastructureAddContainersLogPath from 'images/infrastructure_screenshot-crop_add-containers-log-path.webp'
+
+import logsParsingUi from 'images/infrastructure_screenshot-full_logs-parsing-ui.webp'
+
+Our [infrastructure agent](/docs/infrastructure/infrastructure-monitoring/get-started/get-started-infrastructure-monitoring) can gather logs from your containers. When you enable logs from the infrastructure agent, the agent will automatically assign those logs to the hosts running Docker.
+
+## Requirements [#requirements]
+
+To collect logs from your containers, you need to have:
+
+    * Infrastructure agent [1.8.32](/docs/release-notes/infrastructure-release-notes/infrastructure-agent-release-notes/new-relic-infrastructure-agent-1832) or higher running on Linux
+    * CentOS version 6.0 or higher, if you're using CentOS
+    * Docker with engine from [v1.12](https://docs.docker.com/engine/release-notes/prior-releases/#1120-2016-07-28) or other [`containerd`-based container runtimes](https://kubernetes.io/docs/setup/production-environment/container-runtimes)
+
+## Collect logs from your containers [#collect-logs]
+
+1. Open the configuration file for the infrastructure agent and add this snippet:
+
+    ```yml
+    - name: containers
+      file: /var/lib/docker/containers/*/*.log
+    ```
+
+<img
+    title="Add the containers path to your snippet"
+    alt="A screenshot that shows where to add this snippet in context of your yaml file"
+    src={infrastructureAddContainersLogPath}
+/>
+
+* You may find this file at this path: `/etc/newrelic-infra/logging.d/logging.yml`.
+* This section directs the agent to collect logs from your containers
```

Comment:
We probably don't need to add this. Please note that the Log Forwarder integration picks any YAML file under `/etc/newrelic-infra/logging.d/*.yml` and translates them to Fluent Bit configuration. Contrarily to the public belief, they don't need to be placed under a single `/etc/newrelic-infra/logging.d/logging.yml` file. I believe this missconception was caused by Virtuoso, because that is the file it is using.

In this case, I think it makes sense to just use a separate file to include the Docker container logs config. It results in simpler installation instructions as well, IMHO, as we don't need to tell the user "edit the file in this particular way" (error prone, specially when editing YAML files), but just "create a file with these exact contents".